src/Core/Domain/Tax/Command/BulkDeleteTaxCommand.php

```diff
@@ -50,15 +50,15 @@ public function __construct(array $taxIds)
     }
 
     /**
-     * @return TaxId[]
+     * @return array<TaxId>
```

Comment:
Really? The syntax with pending `[]` is not allowed any more?!

Reply:
It's the same. But it's better, because can be defined.